StudyHub/StudyHub.Common/Utility/ValidationRegexes.cs

```diff
@@ -3,4 +3,5 @@ public static class ValidationRegexes
 {
     public const string FullNameRegex = @"^[A-Za-z\s]*$";
     public const string PasswordRegex = @"^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[#$^+=!*()@%&]).{8,}$";
+    public const string Telegram = @"^@[a-zA-Z0-9_]+";
```

Comment:
![image](https://github.com/fedyafedko/StudyHub/assets/43469966/0ea6a526-044a-47a2-bd39-ef0f8bc64b29)
keep it like it was